README.md

```diff
@@ -52,15 +52,24 @@ Features:
 - [x] Page [middleware](https://nuxtjs.org/docs/2.x/directory-structure/middleware/)
 - [x] Basic [jsx](https://vuejs.org/v2/guide/render-function.html#JSX) (pass `h` as first argument)
 
-Modules:
+Official Modules:
 
 - [X] [http](https://http.nuxtjs.org/)
 - [x] [axios](https://axios.nuxtjs.org/)
 - [X] [i18n](https://i18n.nuxtjs.org/)
 - [x] [content](https://content.nuxtjs.org/)
+- [ ] [auth-next](https://auth.nuxtjs.org/)
 - [ ] Composition API
 - [ ] Tailwindcss
 
+<!--
+Community Modules:
+- [ ] [svg-sprite-module] (https://github.com/nuxt-community/svg-sprite-module)
+-->
+
+Vite Plugins:
```

Comment:
Ideally there should be nuxt modules that support both webpack and vite. Adding vite-only support makes fragmentation in ecosystem and also we still use webpack for production build... So i would suggest not including this list